test/swift_class_test.exs

```diff
@@ -127,34 +137,77 @@ defmodule SwiftClassTest do
   end
 
   describe "class block parser" do
-    input = """
-    "color-" <> color_name do
-      foo(true)
-      color(color_name)
-      bar(false)
-    end
-
-    "color-red" do
-      color(:red)
-    end
-    """
-
-    output = [
-      {{:<>, [context: Elixir, imports: [{2, Kernel}]], ["color-", {:color_name, [], Elixir}]},
-      [
-        ["foo", [true], nil],
-        ["color", [{:color_name, [], Elixir}], nil],
-        ["bar", [false], nil]
-      ]},
-      {
-        "color-red",
-        [
-          ["color", [["IME", "red"]]]
-        ]
-      }
-
-    ]
-
-    # assert parse_class_block(input) == output
+    test "parses a simple block" do
+      input = """
+      defstyle "red-header" do
```

Comment:
I'm using /`defstyle "red-header" do` instead of `"red-header" do` because:
1. Parsing is easier when you can expect keywords to indicate a context
2. It fits better with what one would expect when you see a `do - end` block in Elixir

Reply:
* dropping `defstyle` is a one line change, but my intuition tells me that it should remain. WDYT?

Reply:
`defstyle` needs to go

Reply:
I've removed it